lib/struct.js

```diff
@@ -121,8 +121,10 @@ function Struct () {
   StructType.size = 0
   StructType.alignment = 0
   StructType.indirection = 1
+  StructType.isPacked = arguments[1] || false
```

Comment:
Out of bounds `arguments` checks de-opt the V8 compiler. Do something like this instead:

``` js
arguments.length > 1 ? Boolean(arguments[1] : false
```